packages/card/index.tsx

```diff
@@ -132,10 +132,52 @@ function Description({ children }: DescriptionProps) {
   )
 }
 
+/**
+ * Card variants
+ */
+
+interface PersonCardProps {
+  appearance?: CardProps['appearance']
+  avatar: ThumbnailProps
+  link: string
+  name: string
+  location?: string
+  bio: string
+  productBadges?: ProductBadgesProps['badges']
+}
+
+function PersonCard({
+  appearance,
+  link,
+  avatar,
+  name,
+  location,
+  bio,
+  productBadges,
+}: PersonCardProps) {
+  return (
+    // TODO Determine how to make these two props optional for PersonCard
+    // TODO Possibly make their requirement dependent on whether children are passed or not? May need a type guard function...
+    <Card heading={name} link={link} appearance={appearance}>
+      <Thumbnail {...avatar} />
+      <Content>
+        <Heading>{name}</Heading>
+        {/* ? TODO Possibly extract location to a component for text of this style? */}
+        {location ? <p className={s.location}>{location}</p> : null}
+        <Description>{bio}</Description>
+        {productBadges && productBadges?.length > 0 ? (
+          <ProductBadges badges={productBadges} appearance={appearance} />
+        ) : null}
+      </Content>
+    </Card>
+  )
+}
+
 Card.Thumbnail = Thumbnail
 Card.Meta = Meta
 Card.Content = Content
 Card.Heading = Heading
 Card.Description = Description
+Card.PersonCard = PersonCard
 
 export default Card
```

Comment:
```suggestion

export default Card
export { PersonCard }
```

This would actually be a named export vs adding to the compound component.

Reply:
Ah, it sure would! Good catch :+1: